### ISSUE TITLE
How to Have Variable Substitution for long paths in Geany?

### ISSUE THREAD

**Issue by sami10007** [geany/geany#1057]
I have long paths like in my .tex documents

`/home/masi/Desktop/Mathematics/Exam/Pracitce/Question_set_one_without_answers.pdf
`

which I really would like to have only as $VARIABLE/Filename.pdf. Or anything similar which makes those paths shorter for my ultrabook.

**Comment by elextr** [comment]
That appears to be a question for a tex forum.  Geany has nothing to do with paths you use in your files.

**Comment by sami10007** [comment]
Could there be made an alias? Etc `$MATH` about `$HOME/masi/Desktop/Mathematics/`.

**Comment by b4n** [comment]
you can add snippets, but that will only help you type it, not have it be generic

**Comment by sami10007** [comment]
Yes, snippets do not help because I type already enough fast. Generic feature would only help. Is it technically possible to do it generic?

**Comment by elextr** [comment]
@sami10007 as I said above, its nothing to do with Geany how you encode your filenames in tex.  If tex can support what you are asking for is a question for a tex forum.
